src/Bindings/ManualBindings_Network.cpp

```diff
@@ -99,20 +92,13 @@ static int tolua_cNetwork_CreateUDPEndpoint(lua_State * L)
 	cLuaState::cTableRefPtr callbacks;
 	if (!S.GetStackValues(2, port, callbacks))
 	{
-		LOGWARNING("cNetwork:CreateUDPEndpoint() cannot read its parameters, failing the request.");
-		S.LogStackTrace();
-		S.LogStackValues("Values on the stack");
-		S.Push(false);
-		return 1;
+		return S.ApiParamError("Cannot read parameters");
 	}
 
 	// Check validity:
 	if ((port < 0) || (port > 65535))
```

Comment:
Right, but the error message is misleading. I'll change the `port `variable to `int`, at least for now, and keep the check